operations/helm/charts/mimir-distributed/templates/metamonitoring/recording-rules.yaml

```diff
@@ -0,0 +1,5 @@
+{{- with .Values.metaMonitoring.prometheusRule.recording_rules }}
```

Comment:
This file is going in the right direction. But in this state it will just render the YAML of the rules instead of a kubernetes resource. So helm will try to do something like

```
kubectl apply -f <('
	groups:
	- name: mimir_alerts
	  rules:
	  - alert: MimirIngesterUnhealthy
	    annotations:
	      message: Mimir cluster {{ $labels.cluster }}/{{ $labels.namespace }} has {{
	        printf "%f" $value }} unhealthy ingester(s).
	    expr: |
	      min by (cluster, namespace) (cortex_ring_members{state="Unhealthy", name="ingester"}) > 0
	    for: 15m
	    labels:
	      severity: critical
')
```

which isn't an actual kubernetes manifest. I think an approach similar to what's in 
operations/helm/charts/mimir-distributed/templates/prometheusrule.yaml will be best. You may be able to extend what's already there - not sure what's the tradeoff between having multiple PrometheusRule resources vs having many groups on the same resource

Reply:
I've seen prometheus stack helm charts and my microk8s create multiple CRDs for different purposes , so I think it makes sense to make it separate